README.md

```diff
@@ -1,65 +1,219 @@
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://brambola.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> How to clone and use this project project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+-A working browser application (Google chrome, Mozilla Fire fox, Microsoft edge ...)
+
+-VSCode or any other equivalent code editor
+
+-Node Package Manager (For running, debuging, or installing dependancies to be used)
+
+node and npm:
+
+-Linux
+```sh
+  sudo apt install nodejs npm
+```
+-windows 
+
+[download and install](https://nodejs.org/en/)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- Open the command prompt in the created directory.
+
+- On the terminal run this command 
+```sh
+git clone https://github.com/tsohleDev/brambola.git
+```
+
+- Go to the repository folder using command prompt
+```sh
+ cd brambola
+```
+
+### Install
+
+Install this project with:
+
+```sh
+ run npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+
+```sh
+  npm test
+```
+-
+
+### Deployment
+
+You can deploy this project using:
+
+- To start run 
+```sh
+  npm run build
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> All of the collaborators of this project.
+
+👤 **Tsohle Mokhemisi**
+
+- GitHub: [@githubhandle](https://github.com/tsohleDev2)
+- Twitter: [@twitterhandle](https://twitter.com/RealTsohle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/tsohle-mokhemisi-3687401b2/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- **[Create another page for stocks and change the home page]**
+- **[Change the API to a free one]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> To support this project
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Credits to everyone who inspired the codebase.
+
+I would like to thank [Microverse](https://microverse.org) for the great content
+
```

Comment:
- [ ] Kindly give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).